api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -449,11 +453,12 @@ function dbRewardsReducer(validatorsDictionary, dbRewards) {
 
 function rewardReducer(network, validators, reward, reducers) {
   let parsedRewards = []
+
   Object.entries(reward.validators).forEach((validatorReward) => {
     if (!validators[validatorReward[0]]) return
     const lunieReward = {
-      id: validators[validatorReward[0]].operatorAddress,
-      ...reducers.coinReducer(network, validatorReward[1]),
+      id: validatorReward[0],
+      ...reducers.coinReducer(network, validatorReward[1].toString(10)),
```

Comment:
can we do a `toJSON` or `toHuman` somewhere to prevent these hacks?

Reply:
The problem there is that toHuman() convert amounts to main denom and includes the denom i.e `9.345 DOT`. Same as `toJSON` I think (need to test, talking from memory). I think it's correct to convert the BN to base 10 number in this case.

Reply:
:/ please check it when you find time. this is really hard to read/understand